tests/test_database.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+from os import remove
+from os.path import isfile
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from space_allocator.app import dojo, database, Staff, Fellow, LivingSpace, Office, Base, RoomDB, PersonDB
+
+
+class TestSaveState(unittest.TestCase):
+
+	def setUp(self):
+		self.dojo = dojo
+		self.database = database
```

Comment:
I have imported an already-instantiated dojo. So i don't need to import Dojo and then say self.dojo = Dojo()